benchmarks/sequel/benchmark.rb

```diff
@@ -0,0 +1,42 @@
+require "harness"
+require "securerandom"
+
+Dir.chdir __dir__
+use_gemfile
+
+require "sequel"
+
+# Base Sequel, no plugins
+DB = Sequel.sqlite
+
+DB.create_table :posts do
+  primary_key :id
+  String :title, null: false
+  String :body
+  String :type_name, null: false
+  String :key, null: false
+  Integer :upvotes, null: false
+  Integer :author_id, null: false
+  DateTime :created_at, default: Sequel::CURRENT_TIMESTAMP
+  DateTime :updated_at, default: Sequel::CURRENT_TIMESTAMP
+end
+
+class Post < Sequel::Model; end
+
+50000.times {
+  Post.create(title: SecureRandom.alphanumeric(30),
+              type_name: SecureRandom.alphanumeric(10),
+              key: SecureRandom.alphanumeric(10),
+              body: SecureRandom.alphanumeric(100),
+              upvotes: rand(30),
+              author_id: rand(30))
+}
+
+# heat any caches
+Post.where(id: 1).first.title
+
+run_benchmark(10) do
+  1000.times do |i|
+    Post.where(id: i + 1).first.title
+  end
+end
```

Comment:
I think it would be nice to do a bit more within the benchmark. Maybe retrieve more fields, again thinking in the context of a realistic use case where you would want to display the posts for example.

Reply:
🤦‍♀️ Well, we can probably do a bit better for this benchmark. Maybe concatenate a few of the fields you would normally display in a post. Content, author name, etc?